src/modules/account/index.ts

```diff
@@ -77,7 +77,7 @@ export const Account = {
      * @param {string} pPassword Password of user to login
      * @returns 'true' if login succeeded.
      */
-    async login(pUsername: string, pPassword: string): Promise<boolean> {
+    async login(pUsername: string, pPassword: string, pMetaverseUrl?: string): Promise<boolean> {
```

Comment:
This optional parameter is odd in that the routine checks if the ActiveMetaverse is online. If one passes in a different metaverse url, the account will be logged into the different metaverse but the global metaverse and account information will be different.
If there is some specific use case where this confusing situation is required, that should be explained in the function description.
And, that new parameter is not mentioned in the function API definition.

Reply:
I have updated babylon.js library. The new babylons js library , the functionalities are split up with multiple part.
Like below:
import { Color3, Color4, Vector3 } from "@babylonjs/core/Maths/math";
Vector3 is now part of math that is under core library
So it is not like simply as BABYLON.Vector3 anymore.

Reply:
Also removed pMetaverse optional parameter.

Reply:
https://github.com/vircadia/vircadia-web/pull/43/commits/e72384e8a4860b000f28bb573d67f3cc6cb2cef6

Reply:
removed pMetaverseUrl optional parameter
https://github.com/vircadia/vircadia-web/pull/43/commits/e72384e8a4860b000f28bb573d67f3cc6cb2cef6